#Set time
bool set_time				#Change time
uint8 time_minutes
uint8 time_hours
bool time_24

#Set first time offset
bool set_offset_0 			#Change the offset
int8 offset_0	  			#Offset value in minutes
bool offset_0_inv 			#Invert offset
uint8 offset_0_24 			#24 hour display

#Set second time offset
bool set_offset_1			#Change the offset	
int8 offset_1				#Offset value in minutes
bool offset_1_inv			#Invert offset
uint8 offset_1_24			#24 hour display